users/config.rst

```diff
@@ -436,16 +502,6 @@ Device Element
         <maxRequestKiB>0</maxRequestKiB>
         <remoteGUIPort>0</remoteGUIPort>
     </device>
-    <device id="2CYF2WQ-AKZO2QZ-JAKWLYD-AGHMQUM-BGXUOIS-GYILW34-HJG3DUK-LRRYQAR" name="syno local" compression="metadata" introducer="false" skipIntroductionRemovals="false" introducedBy="">
-        <address>tcp://192.0.2.1:22001</address>
-        <paused>true</paused>
-        <allowedNetwork>192.168.0.0/16</allowedNetwork>
-        <autoAcceptFolders>false</autoAcceptFolders>
-        <maxSendKbps>100</maxSendKbps>
-        <maxRecvKbps>100</maxRecvKbps>
-        <maxRequestKiB>65536</maxRequestKiB>
-        <remoteGUIPort>8384</remoteGUIPort>
-    </device>
```

Comment:
I wouldn't mind having these two examples.  The `<address>` field is explained here, so seeing static and dynamic examples on a first glance is helpful.